### DISCUSSION TITLE
Fall 2022 Student Feedback

### DISCUSSION THREAD

**Discussion by liammulh** [numberscope/frontscope#216] [Ideas]
In today's student meeting, we solicited some feedback from the students. Here's what we got:

- Olivia: I dislike how clicky it is.
  - Ideally one click.
  - Kate: We thought about zero click.
- Olivia: Why don't you just go directly to the big image?
  - Answer: Bundle cards, but ideally we just have one big canvas and maybe a carousel or film strip for other bundles you've made.
- Olivia: Why are the wheels on the main page?
  - Answer: Those are old turtle visualizer spirals.
- Clyde: Thanks for the docsite.
- Clyde: Intro to Git would be nice.
- Olivia: Earlier review would have been nice.
- Olivia: Ordering of visualizers should be easy to difficult.
  - Olivia: Order them in some specific way.
- Kate: Choosing a visualizer...
  - Kate: Button pops up a screen with list of visualizers and descriptions.
  - Glen: Maybe this screen has small thumbnails of visualizations.
- Kate: Gallery page with cool visualizer + sequence + params combinations.
- Clyde: Something on p5 would be nice for onboarding.

**Comment by gwhitney** [comment]
I think the only two ideas we have not (yet?) implemented in ui2 are

 * instituting some sort of ordering(s) on visualizers (and sequences and saved/featured specimens, I suppose)
 * Adding a p5 intro to the docs.

Leaving this discussion open as placeholders for those two ideas, so we can decide whether we want to implement them.
